main.go

```diff
@@ -100,7 +100,7 @@ func mainWithError() (err error) {
 				panic(fmt.Sprintf("%#v\n", microerror.Mask(err)))
 			}
 
-			go newService.Boot(ctx)
+			go newService.Boot(ctx, v.GetBool(f.Service.App.Unique))
```

Comment:
I'm not keen on the wiring here. Can we get the unique bool once and pass it as a setting to Service?

Reply:
I think we can pull `unique` constraint from `config.Viper.GetBool(config.Flag.Service.App.Unique)` directly without parameters, thanks.